### PR TITLE
553 Remove character limit for questions

### PR DESCRIPTION
**Pull request by tillprochaska**

### CODE REVIEW

File: spec/models/question_answer_spec.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 RSpec.describe QuestionAnswer, type: :model do
   describe '#question' do
-    context 'more than 20 characters' do
-      subject { build(:question_answer, question: ("q" * 21)) }
+    context 'more than 640 characters' do
```
Comment (tillprochaska) [inline spec/models/question_answer_spec.rb]: @roschaefer: Out of curiosity: What’s the reason for the 640 characters limit for questions and answers?

Comment (drjakob) [reply spec/models/question_answer_spec.rb]: Facebook

Comment (tillprochaska) [reply spec/models/question_answer_spec.rb]: Ok. So should the character limit be removed completely for both, questions and answers, or should we leave it at 640 characters?

Comment (drjakob) [reply spec/models/question_answer_spec.rb]: Should be completly  removed. (And what happens to our beautiful character counter btw?)

Comment (tillprochaska) [reply spec/models/question_answer_spec.rb]: The character counter was only shown for the main part of the sensor story text. But if you’d like, I could easily add it to questions and/or answers as well.

Comment (drjakob) [reply spec/models/question_answer_spec.rb]: yes, please

**Pull request merged**